mansplainer.py

```diff
@@ -52,9 +52,12 @@ def incoming_sms():
             model = appC.models.get("general-v1.3")
             imageresult = model.predict_by_url(url=image_URL)
 
-            mansplanation = None
-            for tag in imageresult["outputs"][0]["data"]["concepts"]:
-                mansplanation = wiki.getSummaryFromWiki(tag["name"])
+            resultList = imageresult["outputs"][0]["data"]["concepts"]
+            selectedConcept = random.choice(resultList)
```

Comment:
Henry, Don't you think the more the tag closer to the beginning the more accurate it it, instead of choosing randomly?